### DISCUSSION TITLE
[Kavita] Publisher Customization

### DISCUSSION THREAD

**Discussion by zhjihuang** [Kareadita/Kavita#3215] [Ideas]
### Idea Description

Now that publishers are a lot more visible, I think it might be useful to be able to customize their icons and aliases.

Currently, "Marvel" has an icon but "Marvel Entertainment" has no icon. It might be useful to have the ability change and add icons for publishers. An alternative could be to make "Marvel Entertainment" display as "Marvel" with it's icon, but I think being able to add publishers and set their icons would be more useful as many publishers like "Archaia Entertainment", etc. have no icons.

### Idea Category

Web UI

### Duration of Using Kavita

2 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This is something I will not be doing. As stated in the release notes, I am looking to the community to handle it. Images can be added/updated to be much better here: https://github.com/Kareadita/kareadita.github.io

Other users have already started on that effort. You are more than welcome to create an issue and start coordinating the effort or tackling yourself.
